hosts

```diff
@@ -9,7 +9,7 @@
 ## +-------------------------  >>d(' _ ')b<<  -------------------------+
 ## |                                                                   |
 ## |         Project: https://github.com/lennylxx/ipv6-hosts           |
-## |         Update : Tue, 31 Oct 2017 13:55:46 +0800                  |
+## |         Update : 五, 03 11月 2017 03:22:23 +0800                  |

```

Comment:
This is not aligned to the previous version.

Reply:
This is data common bug by locale language set to chinese
Maybe we need forced data format?such as:LC_ALL="C" date